cli/javatests/com/engflow/bazel/invocation/analyzer/consoleoutput/ConsoleOutputTest.java

```diff
@@ -76,21 +72,26 @@ public void formatAnalysisOutputShouldIncludeAnalysisOutputData() {
   }
 
   @Test
-  public void formatAnalysisOutputShouldIncludeAnalysisOutputExceptionMessage() {
+  public void formatFailuresShouldIncludeExceptionMessage() {
     ConsoleOutput consoleOutput = new ConsoleOutput(false, false);
     Exception e = new Exception(EXCEPTION_MESSAGE);
-    SuggestionOutput analysisOutput =
-        SuggestionOutput.newBuilder()
-            .setAnalyzerClassname("Test")
-            .setFailure(
-                SuggestionOutput.Failure.newBuilder()
-                    .setMessage(e.getMessage())
-                    .setStackTrace(Throwables.getStackTraceAsString(e)))
-            .build();
-    String formattedOutput = consoleOutput.formatAnalysisOutput(analysisOutput);
+    var failures =
+        List.of(
+            SuggestionOutput.Failure.newBuilder()
+                .setMessage(e.getMessage())
+                .setStackTrace(Throwables.getStackTraceAsString(e))
+                .build());
+    String formattedOutput = consoleOutput.formatFailures(failures);
     assertThat(formattedOutput).contains(EXCEPTION_MESSAGE);
   }
 
+  @Test
+  public void formatSuggestionOutputCaveatsShouldIncludeCaveatMessage() {
+    ConsoleOutput consoleOutput = new ConsoleOutput(false, false);
+    String formattedOutput = consoleOutput.formatSuggestionOutputCaveats(List.of(CAVEAT));
+    assertThat(formattedOutput).contains(CAVEAT.getMessage());
+  }
+
   @Test
```

Comment:
Suggest adding a test to ensure ordering by benefit is correct.

Reply:
Refactored and added a test. PTAL.